docs/_includes/lhn-template.html

```diff
@@ -0,0 +1,44 @@
+<ul class="lhn-items">
```

Comment:
We're indenting with 2 spaces in this file, we should change it to 4 spaces

Reply:
Fixed